apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations-v2/[evaluationUuid]/editor/_components/EvaluationEditor/index.tsx

```diff
@@ -135,14 +132,14 @@ export function EvaluationEditor({
                   />
                 }
                 metadata={metadata}
-                prompt={evaluation.configuration.prompt}
+                prompt={prompt}
                 onChangePrompt={onChange}
                 freeRunsCount={freeRunsCount}
                 showCopilotSetting={copilotEnabled}
               />
               <TextEditor
                 compileErrors={metadata?.errors}
-                value={evaluation.configuration.prompt}
+                value={value}
```

Comment:
careful with performance here why do you need value? consider if you can use prompt instead

Reply:
true